test/tools/d_do_test.d

```diff
@@ -225,7 +225,7 @@ bool findTestParameter(const ref EnvData envData, string file, string token, ref
         if (close >= 0)
         {
             string[] oss = split(result[1 .. close], " ");
-            if (oss.canFind(envData.os))
+            if (oss.canFind!(o => o.skipOver(envData.os) && (o.empty || o == envData.model)))
```

Comment:
Could you please explain a bit more why the normal find doesn't work and your reasoning for the new condition. Ideally as a comment in the script so that it's clear to future people reading the source code. Thanks!

Reply:
Added an explanation and rationale for `${OS}${MODEL}`.

The current implementation just felt less wasteful than
```oss.canFind(envData.os, envData.os ~ envData.model)```